server/db/create_test_data.js

```diff
@@ -4,14 +4,19 @@
 
 // this script assumes you've run the create_db script.
 
+//jscs:disable
```

Comment:
Ah yes... I remember that line. That would be much easier w/ Knex.

How about jus' escapin' the single quote:
```js
query += '(' + next.map(function(item) { return '\'' + item + '\''; }).join(', ') + ')';
```